main.js

```diff
@@ -92,6 +92,22 @@ function main(args) {
 
     ringpop.setupChannel();
 
+    var shuttingDown = false;
+    process.on('SIGINT', function onSignal(){
+        if (!shuttingDown) {
+            shuttingDown = true;
+            ringpop.selfEvict(function afterSelfEvict(err){
+                if (err) {
+                    console.error('Failure during selfEvict: ' + err);
+                    process.exit(1);
+                    return;
+                }
+                process.exit(0);
+            });
+        } else {
+            process.exit(1);
```

Comment:
I think we should not exit here, or we should log a message that we received a second interrupt and therefore are aborting whatever we are doing and shut down.

What is the reasoning behing the emergency exit here?